webapp/tests/model/test_eligibility_data.py

```diff
@@ -669,7 +635,24 @@ def test_if_user_b_elster_valid_and_pension_yes_then_raise_no_validation_error(s
                 PensionEligibilityData.parse_obj(valid_data)
         except ValidationError as e:
             self.fail("PensionEligibilityData.parse_obj should not raise validation error")
-
+            
+    def test_if_single_user_has_elster_account_and_pension_yes_then_raise_no_validation_error(self):
+        valid_data = {'pension_eligibility': 'yes'}
+        try:
+            with patch('app.model.eligibility_data.SingleUserElsterAccountEligibilityData.__init__', 
+                       MagicMock(return_value=None)):
+                PensionEligibilityData.parse_obj(valid_data)
+        except ValidationError as e:
+            self.fail("PensionEligibilityData.parse_obj should not raise validation error")
+            
+    def test_if_user_b_has_elster_account_pension_yes_then_raise_no_validation_error(self):
+        valid_data = {'pension_eligibility': 'yes'}
+        try:
+            with patch('app.model.eligibility_data.UserBElsterAccountEligibilityData.__init__', 
+                       MagicMock(return_value=None)):
+                PensionEligibilityData.parse_obj(valid_data)
+        except ValidationError as e:
+            self.fail("PensionEligibilityData.parse_obj should not raise validation error")
```

Comment:
Can we add one more test `test_if_joint_taxes_user_a_has_elster_account_pension_yes_then_raise_validation_error`?